cloudfoundry-example/README.md

```diff
@@ -0,0 +1,28 @@
+# Vert.x on Cloud Foundry
```

Comment:
Would you mind to convert this to Asciidoc? There are older examples still on Markdown but let's not add new ones in this format.

Reply:
Yeah I don't normally use Asciidoc, but let me try